### PR TITLE
[WCAG 2.5.8] Update target sizes per new requirements

### LINKED ISSUE #39: Android guidelines - Update target sizes (e.g., buttons) per new requirements
Define and add implementation examples (if applicable) for the given WCAG 2.2 Level A/AA points:

- [ ] [2.5.8](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=258)

Keep in mind that some points are already defined in this repository but it might be that they are still for WCAG 2.1 or they are only written for Android traditional view system. We also need to add compose support where that makes sense.

Your main reference points should be:
1. The [WCAG website](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa) 
2. The iOS implementation of the same point in this repository

**If you find some useful articles, videos, or any resource please share it with the rest of us in our dedicated slack channel.
For any questions you can ask Renato, Petra and Majcen**

### PR DESCRIPTION
**Pull request by zavrskina**
**Description:**

Added changes closes #39 
- introduced a new section for Input modalities including guidelines for minimum target size

Just a note—to me, this guideline seems very dependent on the accessible design, but I decided to add it anyway since the official Android documentation also mentions it. I added a sentence at the end of the section stressing this dependency. 
Feel free to share if you think this section should be removed or have any other ideas or suggestions.

### CODE REVIEW

**Review by mihaelKampic — COMMENTED** [review]
**To answer your question:** 
I would include this section in the guidelines.

**Explanation:**
While material components tend to follow touch target size guidelines, I believe some projects don't use many material components and thus don't meet this criterion.

File: docs/guidelines/platforms/android/guideline_operable_android.md [hunk 1]
```diff
@@ -280,6 +280,56 @@ The example given in the **Screenshot 4.** - **Avoid using ClickableSpan** and i
...
+- In xml layout file:
+```
+<ImageButton ...
+    android:paddingLeft="4dp"
+    android:minWidth="40dp"
+    android:paddingRight="4dp"
+
+    android:paddingTop="8dp"
+    android:minHeight="32dp"
+    android:paddingBottom="8dp" />
```
[the remark this answers is not in the document]
Comment (zavrskina) [reply docs/guidelines/platforms/android/guideline_operable_android.md:310]: If I am not mistaken `minWidth` does not include padding but sets the minimum width of the content area of the view. Padding is added on top of that value. This example was taken from the official docs that states the same: https://developer.android.com/guide/topics/ui/accessibility/apps#large-controls. I did not want to copy-paste these conditions but I might if you think it is needed.

[the remark this answers is not in the document]
Comment (mihaelKampic) [reply docs/guidelines/platforms/android/guideline_operable_android.md:310]: I wasn't aware of this condition 🤯 
Thanks for clarification 👍

[the remark this answers is not in the document]
Comment (mihaelKampic) [reply docs/guidelines/platforms/android/guideline_operable_android.md:310]: Copying the conditions could make it more understandable, but let's hear what others will say on this topic 😄

[the remark this answers is not in the document]
Comment (Aksi0m) [reply docs/guidelines/platforms/android/guideline_operable_android.md:310]: I would add a note here which clarifies that the width/height sum is taken into account the paddings and the min size set for either height/width. Also, referencing the link from the documentation will for sure help.

File: docs/guidelines/platforms/android/guideline_operable_android.md [hunk 2]
```diff
@@ -280,6 +280,56 @@ The example given in the **Screenshot 4.** - **Avoid using ClickableSpan** and i
...
+    modifier = Modifier
+        .padding(start = 4.dp, end = 4.dp, top = 8.dp, bottom = 8.dp)
+        .defaultMinSize(minWidth = 40.dp, minHeight = 32.dp),
+    // Add your onClick or other parameters here
+) {
+    // Content of the IconButton goes here
+}
+```
+More about success criterion can be found on the [official WCAG page](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
+Additionally, implementation details can be found in the [official Android documentation](https://developer.android.com/guide/topics/ui/accessibility/apps#large-controls).
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_operable_android.md:325]: 👍

**Review by Aksi0m — APPROVED** [review]
Looks good! Thanks Petra 🎖️

File: docs/guidelines/platforms/android/guideline_operable_android.md [hunk 3]
```diff
@@ -280,6 +280,56 @@ The example given in the **Screenshot 4.** - **Avoid using ClickableSpan** and i
...
+IconButton(
+    modifier = Modifier
+        .padding(start = 4.dp, end = 4.dp, top = 8.dp, bottom = 8.dp)
+        .defaultMinSize(minWidth = 40.dp, minHeight = 32.dp),
+    // Add your onClick or other parameters here
+) {
+    // Content of the IconButton goes here
+}
+```
+More about success criterion can be found on the [official WCAG page](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_operable_android.md:324]:
```suggestion
More about success criterion, and also some **exceptions** regarding this rule, can be found on the [official WCAG page](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
```

**Pull request merged**